content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: kind
+description: Instructions to setup kind for Istio.
+weight: 21
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare kind for Istio installation.
+
+## Prerequisites
+
+- Please use latest Go version, ideally Go 1.13 or greater.
+- To use kind, you will also need to [install docker](https://docs.docker.com/install/).
+- Install the latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+## Installation steps
+
+1.  Create cluster with the following command:
+
+    {{< text bash >}}
+    $ kind create cluster --name istio-testing
+    {{< /text >}}
+
+    `--name` is used to assign specific name to the cluster. By default, the cluster will be given the name `kind`.
+
+1.  When you want to see the list of kind clusters, use following command:
+
+    {{< text bash >}}
+    $ kind get clusters
+    istio-testing
+    {{< /text >}}
+
+1.  You can also check list of available local cluster contexts using following command.
+
+    {{< text bash >}}
+    $ kubectl config get-contexts
+    CURRENT   NAME                 CLUSTER              AUTHINFO             NAMESPACE
+    *         kind-istio-testing   kind-istio-testing   kind-istio-testing
+              minikube             minikube             minikube
+    {{< /text >}}
+
+    {{< tip >}}
+    `kind` is automatically prefixed in cluster name `kind-istio-testing`
+    {{< /tip >}}
+
+1.  If you run multiple clusters, you need to choose which cluster `kubectl` talks to. You can set a default cluster
+    for `kubectl` by setting the current context in [Kubernetes kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file. Additionally you can run following command
+    to set the current context for `kubectl`.
+
+    {{< text bash >}}
+    $ kubectl config use-context kind-istio-testing
+    Switched to context "kind-istio-testing".
+    {{< /text >}}
+
+    Once you are done setting up local cluster, you can proceed to [setup Istio](/docs/setup/getting-started/#download)
+    on it.
+
+1.  When you are done experimenting and you want to delete existing cluster, use the following command:
+
+    {{< text bash >}}
+    $ kind delete cluster --name istio-testing
+    Deleting cluster "istio-testing" ...
+    {{< /text >}}
+
+## Setup Dashboard UI for kind
+
+kind does not have in built dashboard UI like minikube. But you can still setup and view your cluster.
```

Comment:
does not have ~~in~~/**a** built **in** dashboard UI like minikube.  But you can still setup Dashboard, a web based Kubernetes UI, to view your cluster.

Reply:
Looking at https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/, it seems this is called `Dashboard`. Means my prior remarks about putting an `a` in front were wrong, and we should have just capitalized it.